config/vufind/facets.ini

```diff
@@ -230,9 +230,13 @@ special_facets   = "illustrated,daterange"
 ; language files found in the languages directory. You may add a colon and the
 ; name of a text domain after the field name to specify translations in a specific
 ; text domain (subdirectory of the languages folder). If you comment out the lines
-; below, no facets will be translated.
+; below, no facets will be translated. If you add a second colon, you may add a
+; format descriptor containing the placeholders %raw% and %translated%. This can
+; be useful if the facet is related to a classification like e.g. dewey and
+; you would like to show the raw value together with the translation.
 ;translated_facets[] = institution
 ;translated_facets[] = building
+;translated_facets[] = dewey-raw:DDC23:%raw% - %translated%
```

Comment:
I don't think `dewey-raw` actually makes sense as the example here, since that contains the full call number, not just the first three digits. Maybe it would be better to set up a slightly larger example with a prefixing comment, like:

```
; If you change the default Dewey indexing to omit translation mapping at index time,
; you can uncomment the below configuration to take advantage of on-the-fly
; translation into multiple languages.
;translated_facets[] = dewey-ones:DDC23:%raw% - %translated%
;translated_facets[] = dewey-tens:DDC23:%raw% - %translated%
;translated_facets[] = dewey-hundreds:DDC23:%raw% - %translated%
```